components/Pledge/Merci.js

```diff
@@ -280,15 +278,18 @@ class Merci extends Component {
                 }
               )}
             </H1>
-            <RawHtml
-              type={Lead}
-              dangerouslySetInnerHTML={{
-                __html: t.first([
-                  `merci/lead/package/${query.package || 'UNKOWN'}`,
-                  'merci/lead'
-                ])
-              }}
-            />
+            <Lead>
+              {t.first([
+                `merci/lead/package/${query.package || 'UNKOWN'}`,
+                'merci/lead'
+              ])}
+            </Lead>
```

Comment:
We still seem to use html and therefore need `RawHtml` for many packages:
<img width="999" alt="Screenshot 2021-11-24 at 13 46 18" src="https://user-images.githubusercontent.com/410211/143241291-fbb3e23c-3bb4-4720-b5d8-4cd48125e2b0.png">

Reply:
Ah yeah. I didn't check with the old translations. hate the use of html in the translation strings. makes my head hurt XD